test/integration/dashboards/dashboard.js

```diff
@@ -31,7 +31,8 @@ context('dashboard', function() {
     cy
       .get('.dashboard__frame')
       .find('.dashboard__iframe iframe')
-      .should('have.attr', 'src', '/test_dashboard');
+      .should('have.attr', 'src')
+      .and('include', `https://us-west-2.quicksight.aws.amazon.com/embed/embed_id/dashboards/${ testDashboard.id }?`);
 
```

Comment:
I want to also test that the `<body>` content in the iframe is successfully loaded. But I'm having a hard time getting cypress to see/grab it. So I'm still working on that.

Reply:
I still am not sure how to do this. In cypress, the `<body>` inside the iframe is always empty.

Are we fine with what we have currently? Just testing that the iframe loads with the correct `src` attribute? If not, I can come back to this after we test the implementation on QA.

Reply:
I'm not worried about testing the content too much..  do you know if it does actually show it?

if so I think it's just that `cy.iframe()` helper we added for testing forms?

Reply:
I can see the body content in the test ([screenshot](https://github.com/user-attachments/assets/f6fe580a-d428-49ee-b515-c9252b80e289)).

Yep, I tried `cy.iframe()`. I have no trouble getting the iframe. It's just that the `<body>` content comes back as empty. Cypress can't see it for some reason.

Interestingly, `cy.iframe()` was returning two iframes.

Reply:
ah.. I bet the sdk adds an iframe that we're picking up... fun.. well okie

Reply:
Ya I'm also thinking it's something along those lines...